app-header-layout/app-header-layout.html

```diff
@@ -147,6 +149,32 @@
         this._updateScroller();
       },
 
+      // Resizes app-header-layout to the height of the viewport and translates the content
+      // up to "simulate" disabled document scrolling on iOS.
+      simulateDisabledScroll: function(disabled) {
```

Comment:
It could be "protected" (though we need to decide conventions around protected methods within app-layout elements). Maybe eventually users would want access to this method as well, but it's a good idea to keep it private for now.